src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs

```diff
@@ -100,7 +100,15 @@ internal sealed class ServiceFunction : BuiltinFunction
             }
 
             foreach (var requiredParam in requiredParamInfo)
-                _parameterDescriptionMap.Add(requiredParam.TypedName.Name.Value, requiredParam.Description);
+                if (_parameterDescriptionMap.ContainsKey(requiredParam.TypedName.Name.Value))
+                {
+                    // Can happen when a parameters is in Boby and Query at the same time
+                    _parameterDescriptionMap[requiredParam.TypedName.Name.Value] += $";{requiredParam.Description}";
+                }
+                else
```

Comment:
ServiceFunction comes from PowerApps, and we're trying to stay mirrored to that. 

How does PowerApps handle this situation today? Ie, should this logic be somewhere else (not changing this function, which will hopefully be shared with PA)

Reply:
alternatively, if we do keep this code:
1. wrap in { } 
2. don't repeat 'requiredParam.TypedName.Name.Value' 3 times. 

            foreach (var requiredParam in requiredParamInfo)
            {
                var key = requiredParam.TypedName.Name.Value;
                if (_parameterDescriptionMap.ContainsKey(key))
                {
                    // Can happen when a parameters is in Body and Query at the same time
                    _parameterDescriptionMap[key] += $";{requiredParam.Description}";
                }
                else
                {
                    _parameterDescriptionMap.Add(key, requiredParam.Description);
                }
            }